llvm/test/CodeGen/AMDGPU/insert_extract_element.ll

```diff
@@ -0,0 +1,218 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc -mtriple=amdgcn--amdhsa -mcpu=gfx90a -verify-machineinstrs < %s | FileCheck -check-prefixes=GFX90a %s
```

Comment:
Nit: capital A in GFX90A please

Reply:
Don't need -verify-machineinstrs

Reply:
No, march names are canonically all lowercase

Reply:
Oh, for the check prefix. That should be all caps